src/main/java/at/hannibal2/skyhanni/utils/TabListData.kt

```diff
@@ -107,4 +115,15 @@ class TabListData {
             TabListUpdateEvent(getTabList()).postAndCatch()
         }
     }
+
+    @SubscribeEvent
+    fun onTabListUpdate(event: TabListUpdateEvent) {
+        for (line in event.tabList) {
+            islandNamePattern.matchMatcher(line) {
+                fullyLoaded = true
+                return
+            }
+        }
+        fullyLoaded = false
+    }
```

Comment:
Im not sure if I like this change
A tablist fully loaded would be more fitting in a TablistData object, instead of utils object with a lot of different things

Reply:
I do not care how you resolve it. But very much the same thing is done in HypixelData. So find a soultion where we don't have to do 100% of the same checks to change 2 different variables in 2 places.

Reply:
Oh my god I'm actually stupid
I kept checking in lorenzutils and thought "there isn't any code close enough to this"